soc/riscv/riscv-privilege/telink_b91/CMakeLists.txt

```diff
@@ -13,4 +13,3 @@ zephyr_ld_options(-fuse-ld=bfd)
 # Set compile options
 zephyr_compile_options_ifdef(CONFIG_TELINK_B91_HWDSP -mext-dsp)
 zephyr_compile_options_ifndef(CONFIG_RISCV_GP -mno-relax)
-zephyr_linker_sources(ROM_START init.ld)
```

Comment:
With this change, the `.init.*` sections are now being discarded for this platform (see the map file).

Reply:
I changed it. Added `SORT_KEY 0x0` and now `.init.*` sections come before the vector sections.

Reply:
Confirmed, thanks.